src/components/onboarding/__stories__/selectable_link.story.tsx

```diff
@@ -1,12 +1,12 @@
-import { storiesOf } from "@storybook/react"
-import * as React from "react"
+import { storiesOf } from '@storybook/react';
+import * as React from 'react';
 
-import SelectableLink from "../selectable_link"
+import SelectableLink from '../selectable_link';
```

Comment:
Some prettier/formatting setting is not setup right in your env. @orta do you have any ideas from the top of your head?

Reply:
Is the extension installed in vscode?

Reply:
@orta prettier or typescript lib? Either way I believe I have both?

Reply:
I made a PR #272 which could/should update that